index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const { getAllMovies, getMoviesByTitleOrDir, saveNewMovie } = require('./controller/moviesController')
+
+const app = express()
+
+app.get('/movies', getAllMovies)
+
+app.get('/movies/:input', getMoviesByTitleOrDir)
+
+app.post('/', bodyParser.json(), saveNewMovie)
```

Comment:
Your post doesnt actually post to movies, its not really posting to anything ....
`app.post('/movies', bodyParser.json(), saveNewMovie)`